src/lib/expansion/expansion.md

```diff
@@ -98,3 +98,10 @@ panel can be expanded at a given time:
 
 </md-accordion>
 ```
+
+### Accessibility
+The expansion panel header has `role="button"`. The expansion panel header has attribute
+`aria-controls` with the expansion panel's id as value.
+
+Users can use the keyboard to activate the expansion panel header to switch between expanded state
```

Comment:
Is it worth noting that as the panel headers are buttons, they are able to be navigated to via keyboard as well?